.gitignore

```diff
@@ -3,3 +3,7 @@ node_modules
 *.log
 *.chache
 *-diff.png
+
+packages/swarm-components/lib/
```

Comment:
I'm being explicit about these paths because `lib/` is a directory that should not be gitignored in `swarm-styles` - it's the equivalent to `src/`. Perhaps some future renaming would be nice here, but leaving as is for now.

Reply:
done